#name : tryf 
# --
try {
  $0
}
catch (${1:Exception} ${2:e}) {
  mLogger.error($2);
}
finally {
}

